VisualStudio.gitignore

```diff
@@ -75,6 +75,7 @@ artifacts/
 *.pidb
 *.svclog
 *.scc
+*.txt
```

Comment:
Text files seem important to version control. Could you tell me a bit about why you think they should be excluded?